CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - diskless:
     - fix issues with dnf command in the livenet module (#528)
     - fix issues preventing access to nodes booting a livenet image (#525)
+  - kernel_config: prevent crash if variable is None (#559)
```

Comment:
I would add the variable name: ...if variable ep_kernel_parameters is undefined.

Reply:
Should be ok now.

My deepest apologies for this very long delay, I was totally over loaded.